keel-sql/keel-sql.gradle.kts

```diff
@@ -45,6 +45,7 @@ tasks.getByName<Delete>("clean") {
 
 dependencies {
   implementation(project(":keel-core"))
+  implementation(project(":keel-artifact"))
```

Comment:
This is very undesirable but is here because we make assumptions about Debian and Docker even when reading from the database.

Reply:
How else would we deserialize things?

Reply:
Using polymorphism support in Jackson like we do for all other types with sub-classes.